pulsar-common/src/main/proto/PulsarApi.proto

```diff
@@ -300,6 +300,7 @@ message FeatureFlags {
   optional bool supports_broker_entry_metadata = 2 [default = false];
   optional bool supports_partial_producer = 3 [default = false];
   optional bool supports_topic_watchers = 4 [default = false];
+  optional bool supports_binary_api_get_partitioned_meta_with_param_created_false = 5 [default = false];
```

Comment:
`supports_binary_api_get_partitioned_meta` ?

Reply:
Renamed to `supports_get_partitioned_meta_v2`